packages/server/src/websockets.ts

```diff
@@ -0,0 +1,60 @@
+import { randomUUID } from 'crypto';
+import http from 'http';
+import ws from 'ws';
+import { logger } from './logger';
+import { getRedis } from './redis';
+
+let wsServer: ws.Server | undefined = undefined;
+
+/**
+ * Initializes a websocket listener on the given HTTP server.
+ * @param server The HTTP server.
+ */
+export function initWebSockets(server: http.Server): void {
+  wsServer = new ws.Server({ noServer: true });
```

Comment:
A thought on other options we should consider: `maxPayload` defaults to 100 MiB, which is a ton of data to be tossing around and could leave the server vulnerable to DoS attacks; we should tune this down to maybe 1 MiB or less?

Reply:
Good call.

We have a server config setting for `maxJsonSize` which defaults to `1mb`.  Thoughts on using the same setting?  Convenience of one setting, vs explicitness of multiple settings.

Reply:
I like one setting for now; we can always split out a new setting if necessary to tune this maximum independently in the future